### PR TITLE
doc: small fix: add localtoc to .RST (NCSDK-6534)

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added Local ToC tree to all .RST pages as requested in JIRA NCSDK-6534

Signed-off-by: Francesco Servidio <francesco.servidio@nordicsemi.no>

### CODE REVIEW

**Review by ru-fu — COMMENTED** [review]
Some small comments - waiting for the CI for a real review.
(Maybe a good idea to push the changes to re-trigger CI ...)

File: scripts/shell/ble_console/README.rst [hunk 1]
```diff
@@ -3,18 +3,21 @@
...
+.. contents::
+   :local:
+   :depth: 2
+
+BLE Console is a desktop application that can be used to communicate with an nRF device over *Bluetooth* Low Energy using the :ref:`shell_bt_nus_readme`.
 
 The application supports Linux only and cannot be run on Windows.
 You should run it on a natively installed Linux.
-If you use a virtual machine instead, you might see problems with the Bluetooth LE connection.
+If you use a virtual machine instead, you might see problems with the |BLE| connection.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply scripts/shell/ble_console/README.rst]: Fixed

File: samples/bluetooth/peripheral_uart/README.rst [hunk 2]
```diff
@@ -3,18 +3,20 @@
...
+   :depth: 2
 
+The Peripheral UART sample demonstrates how to use the :ref:`nus_service_readme`.
+It uses the NUS service to send data back and forth between a UART connection and a |BLE| connection, emulating a serial port over |BLE|.
 
 Overview
 ********
 
-When connected, the sample forwards any data received on the RX pin of the UART 1 peripheral to the Bluetooth LE unit.
+When connected, the sample forwards any data received on the RX pin of the UART 1 peripheral to the |BLE| unit.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply samples/bluetooth/peripheral_uart/README.rst]: Fixed

File: scripts/shell/ble_console/README.rst [hunk 3]
```diff
@@ -14,7 +17,7 @@ If you use a virtual machine instead, you might see problems with the Bluetooth
 Requirements
 ************
 
-* Bluetooth LE compliant receiver or dongle
+* |BLE| compliant receiver or dongle
```
Comment (greg-fer) [inline scripts/shell/ble_console/README.rst:20]: Should this one also be fixed?

**Comment by FrancescoSer** [comment]
Closing as CI bugged with ()

**Comment by FrancescoSer** [comment]
New PR is here: https://github.com/nrfconnect/sdk-nrf/pull/3128/commits

**Pull request closed without merging**